bootstrap-joining-demo/aws-api/ec2-tag-based/src/it/scala/akka/cluster/bootstrap/IntegrationTest.scala

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.cluster.bootstrap
+
+import akka.actor.ActorSystem
+import akka.event.Logging
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.HttpRequest
+import akka.management.cluster.{ClusterHttpManagementJsonProtocol, ClusterMembers}
+import akka.stream.{ActorMaterializer, ActorMaterializerSettings}
+import akka.util.ByteString
+import com.amazonaws.services.cloudformation.AmazonCloudFormationClientBuilder
+import com.amazonaws.services.cloudformation.model._
+import com.amazonaws.services.ec2.AmazonEC2ClientBuilder
+import com.amazonaws.services.ec2.model.{DescribeInstancesRequest, Filter, Reservation}
+import org.scalatest.concurrent.PatienceConfiguration.{Interval, Timeout}
+import org.scalatest.concurrent.{Eventually, ScalaFutures}
+import org.scalatest.time.{Seconds, Span, SpanSugar}
+import org.scalatest.{BeforeAndAfterAll, FunSuite, Matchers}
+import spray.json._
+
+import scala.concurrent.{Await, Future}
+
+trait HttpClient {
+
+  implicit val system: ActorSystem = ActorSystem("simple")
+
+  implicit val materializer: ActorMaterializer = ActorMaterializer(ActorMaterializerSettings(system))
+
+  import system.dispatcher
+
+  val http = Http(system)
+
+  def httpGetRequest(url: String): Future[(Int, String)] =
+    http
+      .singleRequest(HttpRequest(uri = url))
+      .flatMap(
+          r =>
+            r.entity.dataBytes
```

Comment:
`.toStrict` is the usual way of doing this

Reply:
@ktoso fixed :-)